libopenage/presenter/presenter.h

```diff
@@ -87,9 +89,9 @@ class Presenter {
 	/**
 	 * Start the presenter and initialize subsystems.
 	 *
-	 * @param debug_graphics If true, enable OpenGL debug logging.
+	 * @param window_settings The settings to customize the display window (e.g. size, display mode, vsync).
 	 */
-	void run(bool debug_graphics = false);
+	void run(renderer::window_settings window_settings = {});
```

Comment:
```suggestion
	void run(const renderer::window_settings window_settings = {});
```

The passed argument should still be const as it is not supposed to be modified by the presenter.

Reply:
fixed it myself, so we can merge :)

Reply:
Thanks!  I got the hint "Clang-Tidy: Parameter 'window_settings' is const-qualified in the function declaration; const-qualification of parameters only has an effect in function definitions" here, so then I removed this `const`.